packages/suite/src/hooks/guide/useGuideKeyboard.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback, useEffect } from 'react';
+
+import { MODAL } from '@suite-actions/constants';
+import { KEYBOARD_CODE } from '@trezor/components';
+import * as guideActions from '@suite-actions/guideActions';
+import { useSelector, useActions } from '@suite-hooks';
+
+export const useGuideKeyboard = () => {
+    const { guideOpen, isModalOpen } = useSelector(state => ({
+        guideOpen: state.guide.open,
+        // 2 types of modals exist. 1. redux 'modal' based, 2. redux 'router' based
+        isModalOpen:
+            state.modal.context !== MODAL.CONTEXT_NONE || state.router.route?.isForegroundApp,
```

Comment:
this selector is repeated on 3 places, maybe time for custom hook or util?

Reply:
this selector is not final yet

Reply:
more conditions to come

Reply:
5dd3f3dc8b87497e02bb5b906e08eacfe10e7534

Reply:
5dd3f3d